### DISCUSSION TITLE
Add support for Google Coral (New AI)

### DISCUSSION THREAD

**Discussion by Grandma-Betty** [djdd87/SynoAI#81] [General]
What a very cool project found! But how is a NAS its mostly weak CPU powerful enough for a deepstack AI? Are there any plans in the future to support Google Coral devices such as the Google Coral USB?

**Comment by djdd87** [comment]
Most modern NAS' are fine for DeepStack. Use the -cpu image.

I've designed the software so it should be possible to switch to other AIs in the future. I'll look into Coral if I get time.

**Reply by Grandma-Betty** [reply]
That'll be awesome, thanks mate!
